src/__tests__/handleAction-test.js

```diff
@@ -195,4 +196,27 @@ describe('handleAction()', () => {
         .to.deep.equal({ number: 3 });
     });
   });
+
+  describe('with invalid actions', () => {
+    it('should throw a descriptive error when the action object is missing', () => {
+      const reducer = handleAction(createAction('ACTION_1'), identity);
+      expect(() => reducer(undefined))
+      // eslint-disable-next-line max-len
+        .to.throw(/The FSA spec mandates an action object with a type. Try using the createAction\(s\) method./);
```

Comment:
I'd like to push back again on the `eslint-disable-next-line max-len` and the use of a regular expression here.

I'd like to use these overrides only where they must be used. In a 20K line production application, I've seen this happen only in two places:

* directly consuming snake-cased fields from the API on the client (`eslint-disable-next-line camelcase`)
* early returns from iterators in `lodash` (`eslint-disable-next-line consistent-return`)

It wouldn't be a rule if we were permissive about bypassing it; the value of the rule comes from a widespread adherence.

***

I also think we should strive for consistency with respect to [similar tests in the repo](https://github.com/acdlite/redux-actions/blob/master/src/__tests__/createActions-test.js#L17). 

To that end, let's test the error class thrown, so callers know what to catch, and the exact string (I don't think the flexibility afforded by using a regular expression object buys us much).